index.html

```diff
@@ -22,10 +22,10 @@
             <a href="#home" class="m-logo">
               <img src="img/firecracker-13@3x.png" alt="Firecracker logo" />
             </a>
-            <a class="sta-hoverOFF" href="#benefits">Benefits</a>
-            <a class="sta-hoverOFF" href="#how_it_works">HOW IT WORKS</a>
-            <a class="sta-hoverOFF" href="#faq">FAQs</a>
-            <a class="sta-hoverOFF" href="#learn_more">Learn more</a>
+            <a class="sta-hoverOFF" href="#benefits"> Benefits </a>
+            <a class="sta-hoverOFF" href="#how_it_works"> How it works </a>
```

Comment:
Thanks for removing the caps, but why do we need spaces around the link text (goes for all the changes that add spaces in links)?

Reply:
Good point